app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -50,9 +42,6 @@ ns.SearchPage = do (ko
       $('#variablesModal').modal('show')
     $('.launch-customize-modal').click ->
       $('#customizeDataModal').modal('show')
-    
-    setTimeout initModal, 2000 if !window.edscportal
-
```

Comment:
Right. It was me... (Although I think setinterval is a better option if we want to keep polling for the right preferences to come back.) I removed it because 1) the call is already synchronized (see an earlier commit in preferences.js.coffee) and 2) put it there requires initialize the sitetour and the preferences twice (one in the $(document).ready block, one in the class).